src/main/webapp/WEB-INF/themes/desktop/sass/vendor/compass/animation/animate/bouncing/_bouncing-entrances.scss

```diff
@@ -0,0 +1,68 @@
+// ---------------------------------------------------------------------------
```

Comment:
Do we really need these "bouncing" animation scss files? I doubt we'll use them anytime soon.